native/src/main/java/io/ballerina/stdlib/os/nativeimpl/SetEnv.java

```diff
@@ -39,6 +39,10 @@ private SetEnv() {
 
     }
 
+    static {
+        StaticComponentContainer.Modules.exportPackageToAllUnnamed("java.base", "java.util");
```

Comment:
With java 17 we cannot use reflections of not exposed modules. either we need to pass --add-opens java.base/java.util=ALL-UNNAMED to java -jar command or programmatically do that using burningwave library